src/index.html

```diff
@@ -11,7 +11,144 @@
+            name="surname" 
+            autocomplete="off" 
+            required 
+          >
+        </label>
+        <label class="input">
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            autocomplete="off" 
+            required 
+          >
+        </label>
+        <label class="input">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+            required
+          >
+        </label>
+        <label class="input">
+          Full date of birth:
+          <input type="date" name="born">
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input 
+            type="checkbox"
+            name="term"
+            required
+          >
+        </label>
+      </fieldset>
+      <!-- Second field -->
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <label class="input">
+          E-mail:
+          <input
+            type="email"
+            name="mail"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label>
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+      <!-- Third field -->
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+        <label class="input">
+          Do you love cats?
+          <input 
+            type="radio" 
+            name="pet" 
+            value="yes"
+          >
+            Yes
+          <input 
+            type="radio" 
+            name="pet" 
+            value="no"
+          >
+            No
+        </label>
+        <label class="input">
+          What is your favorite color?
+          <input type="color" name="favcolor">
+        </label>
+        <label class="input">
+          What time do you go to bad?
+          <input type="time" name="sleepTime">
+        </label>
+        <label class="input">
+          What are your favorite brand of cars?
+          <select name="car" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label>
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="workRate" 
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <!-- Fourth field -->
+      <fieldset class="field">
+        <legend>Additional info:</legend>
+        <label class="input">
+          Comments:
+          <textarea 
+            name="comments" 
+            cols="15" 
+            rows="2"
+          >
+          </textarea>
+        </label>
```

Comment:
```suggestion
          ></textarea>
        </label>
```

для коректної роботи `textarea` слід прибрати перенос та пробіли між відкриваючим та закриваючим тегами
https://user-images.githubusercontent.com/102413088/189628903-06aafba6-86e6-4e6c-acb3-ca03c2ff9071.mov